src/Proxy/OpcacheProxy.php

```diff
@@ -134,8 +134,11 @@ public function opcache_invalidate($script, $force = false)
      */
     public function opcache_reset()
     {
+        // Avoid using the opcache_reset() return value to workaround PHP bug
+        // https://bugs.php.net/bug.php?id=71621
         $code = new Code();
-        $code->addStatement('return opcache_reset();');
+        $code->addStatement('opcache_reset();');
+        $code->addStatement('return true;');
```

Comment:
Can you add a comment here to the bug?

Reply:
It is exists just a couple lines above this one. Is that sufficient? Or are you suggesting I move it down to this exact line? I'm fine either way, let me know.

Reply:
I missed the comment, all good!